docs/participant_endpoint_enums.md

```diff
@@ -0,0 +1,37 @@
+# Participant Endpoint Enums
+
+This is the list of endpoints that a PISP or DFSP must register with the swith to get callbacks from the `thirdparty-api-adapter` and `transaction-request-service`.
+
+These endpoints should be added to the enums of `central-services-shared`, and the seeds of `central-ledger`
+
+## PISP:
+
+- `THIRDPARTY_CALLBACL_URL_CONSENT_REQUEST_PUT`
+- `THIRDPARTY_CALLBACK_URL_CONSENT_REQUEST_PUT_ERROR`
+- `THIRDPARTY_CALLBACK_URL_CONSENT_POST`
+- `THIRDPARTY_CALLBACK_URL_CONSENT_REQUEST_PUT`
+- `THIRDPARTY_CALLBACK_URL_TRANSACTION_REQUEST_PUT`
+- `THIRDPARTY_CALLBACK_URL_TRANSACTION_REQUEST_PUT_ERROR`
+- `FSPIOP_CALLBACK_URL_TRX_REQ_SERVICE` 
```

Comment:
can I also clarify which endpoints this enum is associated with?

Reply:
Yeah this is an existing enum used for the transactionRequest service, which currently handles the `authorizations` resource. It's an existing service, which is why the name doesn't fit the other conventions.